manager/manager.py

```diff
@@ -598,7 +598,7 @@ def generate_dashboard(self, max_failures=MAX_FAILURES):
                 '<th class="hdr">Repo</th>'
                 '<th class="hdr">PreConvert</th>'
                 '<th class="hdr">Converted</th>'
-                '<th class="hdr">Destination</th>'
+                '<th class="hdr">Destination</th>',
```

Comment:
It would be much easier to understand what is intended here if you used a multi-line string instead of implicit string concatenation. That's probably why someone removed the comma.